modules/api/src/main/scala/fi/spectrum/api/v1/endpoints/HistoryEndpoints.scala

```diff
@@ -26,11 +26,20 @@ final class HistoryEndpoints {
   def orderHistoryE: Endpoint[Unit, (Paging, TimeWindow, HistoryApiQuery), HttpError, OrderHistoryResponse, Any] =
     baseEndpoint.post
       .in(PathPrefix / "order")
-      .in(paging)
+      .in(paging(50))
       .in(timeWindow)
       .in(jsonBody[HistoryApiQuery])
       .out(jsonBody[OrderHistoryResponse])
       .tag(Group)
       .name("Orders history")
       .description("Provides orders history with different filters by given addresses")
+
+  def addressesHistoryE: Endpoint[Unit, Paging, HttpError, AddressesHistoryResponse, Any] =
+    baseEndpoint.get
+      .in(PathPrefix / "addresses")
+      .in(paging(100))
+      .out(jsonBody[AddressesHistoryResponse])
+      .tag(Group)
+      .name("Addresses history")
+      .description("Provides list of addresses that interacted with platform")
```

Comment:
`with amm dex platform` or something like this cause we don't provide data for YF here